tiledb/cloud/notebook.py

```diff
@@ -67,7 +67,8 @@ def download_notebook_to_file(
     ipynb_file_contents = download_notebook_contents(
         tiledb_uri,
     )
-    with open(ipynb_file_name, "w") as handle:
```

Comment:
This is fine as-is, IMO -- sorry about the distraction with the earlier comment. I missed that this is a local FS operation only.